docs/reference/modules/threadpool.asciidoc

```diff
@@ -142,14 +142,23 @@ threadpool:
 [[processors]]
 === Processors setting
 The number of processors is automatically detected, and the thread pool
-settings are automatically set based on it. Sometimes, the number of processors
-are wrongly detected, in such cases, the number of processors can be
-explicitly set using the `processors` setting.
+settings are automatically set based on it. However, the number of
```

Comment:
@bleskes That gets very expert very quickly because to do it right we would also be mentioning topics like tuning the number of GC threads, and considerations like taskset (completely putting aside topics like cgroups and the like too).